integration/spark/app/src/test/java/io/openlineage/spark/agent/DatabricksUtils.java

```diff
@@ -151,13 +161,15 @@ static List<RunEvent> runScript(WorkspaceClient workspace, String clusterId, Str
     runSubmitTaskSettings.setTaskKey(taskName);
     runSubmitTaskSettings.setExistingClusterId(clusterId);
     runSubmitTaskSettings.setSparkPythonTask(task);
+    runSubmitTaskSettings.setTimeoutSeconds(60L);
 
     SubmitRun submitRun = new SubmitRun();
     submitRun.setRunName(taskName);
+    submitRun.setTimeoutSeconds(60L);
     submitRun.setTasks(Collections.singletonList(runSubmitTaskSettings));
 
     // trigger one time job
-    workspace.jobs().submit(submitRun).get();
+    workspace.jobs().submit(submitRun).get(Duration.ofMinutes(3));
```

Comment:
Why are there 3 minutes but previous timeout is 60s?

Reply:
I am not sure if this was necessary or just remained like this when I was doing some tests. Anyway, this does not make succeeding tests go longer.